deploy/test/exchange/ExchangeV2.test.js

```diff
@@ -57,6 +57,49 @@ contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
     await exchangeV2.setProtocolFee(300);
   });
 
+  describe("gas estimation direct Buy/AcceptBid", () => {
```

Comment:
we should add more tests, not only gas estimation
positive/negative test. like does it work in every case it's expected to work? does it not work when it's not expected to?

Reply:
add tests to check balances after direct purchase/accept